scripts/g.extension/g.extension.py

```diff
@@ -159,7 +159,20 @@
 # temp dir
 REMOVE_TMPDIR = True
 PROXIES = {}
+HEADERS = {
+    "User-Agent": "Mozilla/5.0",
+}
+
+
+def urlretrieve_(url, filename):
+    request = Request(url, headers=HEADERS)
```

Comment:
I don't see Request defined anywhere. Am I missing something?

Reply:
I forgot to import it. I fixed it.